src/scheduled_callbacks.toit

```diff
@@ -0,0 +1,139 @@
+// Copyright (C) 2024 Toitware ApS. All rights reserved.
+// Use of this source code is governed by an MIT-style license that can be
+// found in the LICENSE file.
+
+import monitor
+
+class PriorityQueueElement:
```

Comment:
I'd reorder the classes so it starts with the one you want to worry about:

1. ScheduledCallbacks
2. PriorityQueue
3. PriorityQueueElement

I'd also consider making (2) and (3) private.

Reply:
done. (in a separate commits, so it's easier to diff/review)